eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -87,10 +93,44 @@ stages:
       useMonoRuntime: false              # 🚫
       withPreviousSDK: false             # 🚫
 
-  - ${{ if ne(variables['Build.Reason'], 'PullRequest') }}:
+  - template: ../jobs/vmr-build.yml
+    parameters:
+      # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+      buildName: CentOSStream8_Online_PreviousSourceBuiltSdk
+      isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+      vmrBranch: ${{ variables.VmrBranch }}
+      architecture: x64
+      pool:
+        name: ${{ variables.defaultPoolName }}
+        demands: ${{ variables.defaultPoolDemands }}
+      container: ${{ parameters.centOSStream8Container }}
+      buildFromArchive: false            # 🚫
+      enablePoison: false                # 🚫
+      excludeOmniSharpTests: true        # ✅
+      runOnline: true                    # ✅
+      useMonoRuntime: false              # 🚫
+      withPreviousSDK: true              # ✅
 
-    # CI - Stage 1 x64 legs ------------------------------------
+  - template: ../jobs/vmr-build.yml
+    parameters:
+      # Changing the build name requires updating the referenced name in the source-build-sdk-diff-tests.yml pipeline
+      buildName: Alpine317_Offline_PreviousSourceBuiltSdk
+      isBuiltFromVmr: ${{ parameters.isBuiltFromVmr }}
+      vmrBranch: ${{ variables.VmrBranch }}
+      architecture: x64
+      pool:
+        name: ${{ variables.defaultPoolName }}
+        demands: ${{ variables.defaultPoolDemands }}
+      container: ${{ parameters.alpine317Container }}
+      buildFromArchive: false            # ✅
+      enablePoison: true                 # ✅
```

Comment:
Discussed with @NikolaMilosavljevic, and he informed me that running poison on one leg would be sufficient. I'm currently waiting for [this issue](https://github.com/dotnet/source-build/issues/3635) to be resolved so that the Alpine leg can run successfully and indicate the poison results.

Reply:
[Alpine Stage 1 leg](https://github.com/dotnet/installer/blob/e51a67e0c0aada67b320ce21b78a3ecd49ce8188/eng/pipelines/templates/stages/vmr-build.yml#L127) now runs poison and [fedora Stage 1 leg does not](https://github.com/dotnet/installer/blob/e51a67e0c0aada67b320ce21b78a3ecd49ce8188/eng/pipelines/templates/stages/vmr-build.yml#L243)